chainerx_cc/chainerx/python/testing/device_buffer.cc

```diff
@@ -59,7 +59,7 @@ void InitChainerxDeviceBuffer(pybind11::module& m) {
               Dtype dtype = python_internal::GetDtype(dtype_handle);
               int64_t item_size = GetItemSize(dtype);
               int64_t bytes = item_size * total_size;
-              std::shared_ptr<void> host_data = std::make_unique<uint8_t[]>(bytes);
+              std::shared_ptr<void> host_data(new uint8_t[bytes], std::default_delete<uint8_t[]>());
```

Comment:
Could you prefer list initialization, for consistency.
```suggestion
              std::shared_ptr<void> host_data{new uint8_t[bytes], std::default_delete<uint8_t[]>()};
```